pysqldb3/tests/test_io.py

```diff
@@ -221,44 +221,210 @@
-#
+
 #         # Assert df equality -- some types need to be coerced from the Pandas df for the equality assertion to hold
-#         pg_df = db.dfquery("""
+#         pg_df = db.dfquery(f"""
 #         select *
-#         from working.{}
-#         """.format(test_pg_to_sql_table)).infer_objects()
-#
+#         from {pg_schema}.{test_pg_to_sql_table}
+#         """).infer_objects()
+
 #         # Assert
 #         shared_non_geom_cols = list(set(pg_df.columns).intersection(set(spatial_df.columns)) - {'geom'})
 #         pd.testing.assert_frame_equal(pg_df[shared_non_geom_cols], spatial_df[shared_non_geom_cols],
 #                                       check_dtype=False,
 #                                       check_exact=False,
-#                                       check_less_precise=True,
 #                                       check_datetimelike_compat=True)
-#
+
 #         shared_non_geom_cols = list(set(pg_df.columns).intersection(set(spatial_df.columns)) - {'geom'})
 #         pd.testing.assert_frame_equal(pg_df[shared_non_geom_cols], not_spatial_df[shared_non_geom_cols],
 #                                       check_dtype=False,
 #                                       check_exact=False,
-#                                       check_less_precise=True,
 #                                       check_datetimelike_compat=True)
-#
+
 #         # Cleanup
 #         sql.drop_table(schema=sql.default_schema, table=dest_name_not_spatial)
 #         sql.drop_table(schema=sql.default_schema, table=dest_name)
-#         db.drop_table(table=test_pg_to_sql_table, schema=schema)
-#
+#         db.drop_table(table=test_pg_to_sql_table, schema=pg_schema)
+
 #     def test_pg_to_sql_error(self):
 #         return
-#
+
 #     # Note: temporary functionality will be tested separately!
 #     # Still to test: LDAP, print_cmd
-#
+
 #     @classmethod
 #     def teardown_class(cls):
 #         helpers.clean_up_test_table_pg(db)
-#
-#
+
+
+class TestPgtoSqlQry:
+
+    @classmethod
+    def setup_class(cls):
+        helpers.set_up_test_table_pg(db)
+
+    def test_pg_to_sql_qry_basic_table(self):
+
+        db.drop_table(schema=pg_schema, table=test_pg_to_sql_qry_table)
+        assert not db.table_exists(schema = pg_schema, table = test_pg_to_sql_qry_table)
+        sql.drop_table(schema = sql_schema, table = test_pg_to_sql_qry_table)
+        assert not sql.table_exists(schema = sql_schema, table = test_pg_to_sql_qry_table)
+        
+
+        db.query(f"""
+                    create table {pg_schema}.{test_pg_to_sql_qry_table} (test_col1 int, test_col2 int);
+                    insert into {pg_schema}.{test_pg_to_sql_qry_table} VALUES(1, 2);
+                    insert into {pg_schema}.{test_pg_to_sql_qry_table} VALUES(3, 4);
+        """)
+
+        # pg_to_sql_qry
+        data_io.pg_to_sql_qry(db, sql, query=
+                             f"""
+                             select * from {pg_schema}.{test_pg_to_sql_qry_table}
```

Comment:
maybe make the select something more specific than select * with no filter to really ensure its tested correctly